src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/component/ProgressComponent.java

```diff
@@ -48,6 +48,9 @@ public ProgressComponent(ItemStack ico, Text d, Text b, float pcnt, int color) {
     public ProgressComponent(ItemStack ico, Text text, float pcnt, int color) {
         this(ico, text, Text.of(pcnt + "%"), pcnt, color);
     }
+    public ProgressComponent(ItemStack ico, Text text, float pcnt, int max, int color) {
```

Comment:
Empty line between methods here.
```suggestion

    public ProgressComponent(ItemStack ico, Text text, float pcnt, int max, int color) {
```